metaflow/plugins/aws/batch/batch_client.py

```diff
@@ -282,6 +287,29 @@ def _register_job_definition(
                     {"sourceVolume": name, "containerPath": host_path}
                 )
 
+        if use_tmpfs or tmpfs_size:
```

Comment:
we could change this to
```
if use_tmpfs or (tmpfs_size and use_tmpfs != False):
```
it is confusing that `@batch(use_tmpfs=False, tmpfs_size=10000)` uses `tmpfs` even though it says `False`